src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckInDetail/TraceLocationDetailViewController.xib

```diff
@@ -21,6 +21,7 @@
                 <outlet property="bottomCardView" destination="Ek6-qj-gRz" id="zPW-8o-d4S"/>
                 <outlet property="checkInButton" destination="LzL-ri-WCR" id="nhq-cH-UN3"/>
                 <outlet property="checkInForLabel" destination="qe2-Cr-Hsf" id="2Cc-Rz-KNx"/>
+                <outlet property="checkOutView" destination="GDB-JB-hDV" id="k1Q-bf-2zE"/>
```

Comment:
Why do we need this ? we can't we just set the accessibility identifier inside the StackView ?

Reply:
No we can't. We need outlets for checkOutView and switchView (see below) in order to differentiate them.